linked_list/implementation.py

```diff
@@ -1,40 +1,77 @@
 from .interface import AbstractLinkedList
-
+from .node import Node
 
 class LinkedList(AbstractLinkedList):
     """
     Implementation of an AbstractLinkedList inteface.
     """
 
-    def __init__(self, elements=None):
-        pass
+    def __init__(self, elements=[]):
+        self.start, self.end = None, None
+        
+        for elem in elements:
+            self.append(elem)
 
     def __str__(self):
-        pass
+        return '[' + ", ".join([ str(item.elem) for item in self ]) + ']'
```

Comment:
We used 
`return str([elem for elem in self]) `

I think that would also work here as well?